Library/Formula/sdedit.rb

```diff
@@ -0,0 +1,16 @@
+class Sdedit < Formula
+  desc "Tool for generating sequence diagrams very quickly."
+  homepage "http://sdedit.sourceforge.net"
+  url "https://downloads.sourceforge.net/projects/sdedit/files/sdedit/4.0/sdedit-4.01.jar"
+  sha256 "86f52193ceb3a478fd0cb356f631b6131bc649561fcc5effe73edfb3b12c33c9"
```

Comment:
This should be: `49db33ac79775608bcba439eb53a84d59d6906d6539528f6d58a4e8601f741c5`